beacon_node/lighthouse_network/src/service/mod.rs

```diff
@@ -1247,6 +1250,9 @@ impl<AppReqId: ReqId, TSpec: EthSpec> Network<AppReqId, TSpec> {
                         );
                         Some(event)
                     }
+                    InboundRequest::LightClientBootstrap(_) => {
+                        unreachable!()
+                    }
```

Comment:
If I'm reading this right, this is an inbound `LightClientBootstrap` request that we should respond to, isn't it?

I'm not sure what we should do in this case, maybe just return `None`? cc @pawanjay176